--- MavLinkMessageBase_orig.hpp	2017-05-14 20:59:05.732929670 -0400
+++ MavLinkMessageBase.hpp	2017-05-14 20:58:57.976945286 -0400
@@ -53,7 +53,7 @@
 
 		// unpack the given message
 		void decode(const MavLinkMessage& msg);
-        // pack this message into given message buffer 
+        // pack this message into given message buffer
         void encode(MavLinkMessage& msg) const;
 
 		// find what type of message this is and decode it on the heap (call delete when you are done with it).
@@ -79,7 +79,7 @@
 		void pack_uint16_t_array(int len, char* buffer, const uint16_t* field, int offset) const;
 		void pack_int16_t_array(int len, char* buffer, const int16_t* field, int offset) const;
 		void pack_float_array(int len, char* buffer, const float* field, int offset) const;
-		
+
 		void unpack_uint8_t(const char* buffer, uint8_t* field, int offset);
 		void unpack_int8_t(const char* buffer, int8_t* field, int offset);
 		void unpack_int16_t(const char* buffer, int16_t* field, int offset);
@@ -102,7 +102,7 @@
 		std::string int16_t_array_tostring(int len, const int16_t* field);
 		std::string uint16_t_array_tostring(int len, const uint16_t* field);
 		std::string float_array_tostring(int len, const float* field);
-		std::string float_tostring(float value); 
+		std::string float_tostring(float value);
 	};
 
 	// Base class for all strongly typed MavLinkCommand classes defined in MavLinkMessages.hpp
@@ -130,7 +130,7 @@
 	class MavLinkTelemetry : public MavLinkMessageBase {
 	public:
 		const static uint8_t kMessageId = 204; // in the user range 180-229.
-		MavLinkTelemetry() { msgid = kMessageId; }
+		MavLinkTelemetry() : wifiInterfaceName(nullptr) { msgid = kMessageId; }
 		uint32_t messagesSent;		 // number of messages sent since the last telemetry message
 		uint32_t messagesReceived;	 // number of messages received since the last telemetry message
 		uint32_t messagesHandled;	 // number of messages handled since the last telemetry message
